packages/fxa-settings/src/pages/ResetPasswordConfirmed/index.test.tsx

```diff
@@ -23,20 +23,18 @@ describe('ResetPasswordConfirmed', () => {
       'You’re now ready to use Account Settings'
     );
     const passwordResetContinueButton = screen.queryByText('Continue');
-
+    // Calling `getByText` will fail if these elements aren't in the document,
+    // but we test anyway to make the intention of the test explicit
     expect(passwordResetContinueButton).not.toBeInTheDocument();
     expect(passwordResetConfirmation).toBeInTheDocument();
     expect(serviceAvailabilityConfirmation).toBeInTheDocument();
   });
 
   it('emits the expected metrics on render', async () => {
```

Comment:
Nit, I don't think this or some of the others need to be `async`.

Reply:
oh great thinking 👍